src/libcmis/gdrive-document.cxx

```diff
@@ -69,35 +69,39 @@ string GDriveDocument::getDownloadUrl( string streamId )
     if ( !streamId.empty( ) )
     {
         // Find the rendition associated with the streamId
-        for ( vector< RenditionPtr >::iterator it = renditions.begin( ) ; 
-            it != renditions.end(); ++it )
+        for (const auto& renditionPtr : renditions)
```

Comment:
Please try to follow the coding style of the surrounding code. (I don't like it very much myself, but, on my list, consistency is higher than personal preference.)

Reply:
Not sure to understand here, do you mean I should use:
for (const auto& renditionPtr
        : renditions)
?

Reply:
No, that doesn't make much sense. What I mean is: try to use the same indentation, spaces around stuff, etc. like the surrounding code. E.g.,
```c++
if ( renditionPtr->getStreamId( ) == streamId )
```
instead of
```c++
if (renditionPtr->getStreamId() == streamId)
```

Reply:
Ok I added spaces on all locations I had changed.
Hope it's ok now.